src/stackedBars.ts

```diff
@@ -77,8 +79,16 @@ export function renderStackedBars(pareto: Pareto, settings: Settings) {
             .attr("width", bBox.width + 2 * padding)
             .attr("stroke", "#000")
             .attr("stroke-width", settings.style.onMouseOverBox.strokeWidth)
+            .attr("shape-rendering", "crispEdges")
             .attr("fill", "none");
     }
+
+    rectangularSelection({
+        clearMarking: settings.clearMarking,

```

Comment:
I get this error here, is it the same for you? 

![image](https://user-images.githubusercontent.com/63287104/207054884-bf563598-644c-453b-9fb4-b1849abbc427.png)

Reply:
Did you pull the latest code? It should've been fixed in the last commit :)

Reply:
Seems like I didn't, works now thanks!